frontend/src/components/CreateCollectionModal/components/Content/index.tsx

```diff
@@ -107,7 +115,21 @@ const Content: FC<Props> = (props) => {
 
   // Null / tombstone checking is type safety netting.  We shouldn't be getting to these lines/cases since we can't open the modal if the collection is tombstoned/doesn't exist.
   if (isTombstonedCollection(data)) data = null;
-  const { name, description, contact_email, contact_name } = data || {};
+  const {
+    name,
+    description,
+    contact_email,
+    contact_name,
+    consortia: collectionConsortia,
+  } = data || {};
+
+  const [consortia, setConsortia] = useState<DefaultDropdownMenuOption[]>([]);
```

Comment:
instead of `useState` maybe `useMemo` would do?

e.g.,

```ts
const consortia = useMemo(() => {
  if (collectionConsortia) {
    setConsortia(buildConsortiaOptions(collectionConsortia));
  }

  return [];
}, [collectionConsortia])
```

Reply:
Yes wow @tihuan thank you so much!